sql/core/src/test/resources/sql-tests/inputs/literals.sql

```diff
@@ -27,6 +27,12 @@ select 9223372036854775807L, -9223372036854775808L;
 -- out of range long
 select 9223372036854775808L;
 
+-- big decimal parsing
```

Comment:
nit: if we move the two new queries to the end of this file, the following diff of `literals.sql.out` can be less.